concrete/src/File/Image/Svg/Sanitizer.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Concrete\Core\File\Image\Svg;
+
+use DOMDocument;
+use DOMElement;
+use Exception;
+use Throwable;
+
+class Sanitizer
+{
+    /**
+     * Sanitize a file containing a SVG document.
+     *
+     * @param string $inputFilename The input filename
+     * @param \Concrete\Core\File\Image\Svg\SanitizerOptions $options the sanitizer options (if NULL, we'll use the default ones)
+     * @param string $outputFilename The output filename (if empty, we'll overwrite $inputFilename)
+     * @param throws \Concrete\Core\File\Image\Svg\SanitizerException in case of errors
```

Comment:
This should be an `@throws`, not an `@param` and it should be separated from the `@param` group with a blank line